projects/components/src/form/form-select/form-select.component.spec.ts

```diff
@@ -83,6 +92,20 @@ describe('FormSelectComponent', () => {
             expect(selectInput.value).toEqual(hostComponent.options[2].value as string);
         });
     });
+
+    describe('validation', () => {
+        it('should not show error icon before user selects value', () => {
```

Comment:
Please use `does not show error` instead of `should not show error icon`. We like to avoid repetitive text